docs/proof-of-stake/ticket-splitting.md

```diff
@@ -0,0 +1,112 @@
+
+Coordination of sessions is currently done on Decred's Slack group in the #ticket_splitting channel, or in the Telegram group located [here](https://t.me/dcrtktsplit). 
+
+To prevent ticket splitting from being used to game Decred's PoS voting mechanism, voting rights for a split ticket are assigned pseudorandomly to a single participant, based on the  percentage of their contribution. I.e., if someone purchases 60% of a split ticket, they have a 60% chance of being selected as the voter for that ticket. Further details about the voting algorithm can be found [here](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/voter-selection-ago.md). 
+
+### Further reading
+
+For more on ticket splitting, including further explanation of the risks involved, voting, and technical specs of the Split Ticket Buyer software, see the following:
+
+- [DCR Ticket Splitting — All you need to know!](https://medium.com/decred/dcr-ticket-splitting-all-you-need-to-know-b8edc6b65db3): a thorough overview of how ticket splitting works. 
+- [Split Ticket Buyer documentation](https://github.com/matheusd/dcr-split-ticket-matcher/tree/master/docs): Technical documentation for the Split Ticket Matcher software used to fascilitate ticket splitting. 
+
+
+
+## Setting up Ticket Splitting
+
+These instructions will walk you through buying a ticket using the GUI version of Split Ticket Buyer. For instructions on installing the Command-line Interface (CLI) version, see [here](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/client-cli.md).
+
+1. Create an account with a Voting Service Provider (VSP) that supports ticket splitting. Below is a list of VSPs that support ticket splitting.
+
+    - [https://stake.decredbrasil.com](https://stake.decredbrasil.com)
+    - [https://decredvoting.com](https://stake.decredbrasil.com)
+
+1. Login to your VSP and navigate to the `Settings` tab. Copy the API token.
+
+    ![VSP Settings page](../img/ticket_splitting_settings.png)
+
+1. Open your Decrediton wallet. Navigate to the `Tickets` tab, select your VSP from the Stakepool dropdown menu, and paste in your VSP API key. If you already have another VSP configured, add the new VSP as you would any other VSP. 
+
+    ![Ticket tab Decrediton](../img/ticket_splitting_decrediton.png)
+
+1. Enter your wallet passphrase. You should now see your configured stakepool.
+
+    ![Configured Decrediton Stakepool](../img/ticket_splitting_config_stakepool.png)
+
+1. Download the Split Ticket Matcher software for your OS.
+
+    - Split Ticket Matcher [downloads](https://github.com/matheusd/dcr-split-ticket-matcher/releases/tag/v0.7.0).
```

Comment:
Should probably link to the "releases" page instead of a specific tag.

Reply:
Done